js/main.js

```diff
@@ -0,0 +1,55 @@
+const NAMES = [
+  'Андрей','Мария',
+  'Станислав',
+  'Олег'
+];
+const COMMENTS = [
+  'Всё отлично!',
+  'В целом всё неплохо. Но не всё. Когда вы делаете фотографию, хорошо бы убирать палец из кадра.',
+  'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.'
+];
+const DESCRIPTIONS = [
+  'Это мяч',
+  'Коты наше все',
+  'Анапа 2008',
+  '#Я в коридоре на Балли'
+];
+const getRandomPositiveInteger = (a, b) =>{
+  const lower = Math.ceil(Math.min(a, b));
+  const upper = Math.floor(Math.max(a, b));
+  const result = Math.random() * (upper - lower + 1) + lower;
+  return Math.floor(result);
+};
+const createIdGenerator = () => {
+  let lastGeneratedId = 0;
+  return function (){
+    lastGeneratedId += 1;
+    return lastGeneratedId;
+  };
+};
+
+
+const getRandomArrayElement = (array) => array[getRandomPositiveInteger(0, array.length - 1)];
+
+const createComments = () =>({
+  id: getRandomPositiveInteger(15,200),
+  avatar: `img/avatar-${getRandomPositiveInteger(0, 6)}.svg`,
+  message: getRandomArrayElement(COMMENTS),
+  name: getRandomArrayElement(NAMES),
+});
+const createObj = () =>{
+  const randomIdNumber = createIdGenerator();
+  const randomUrlNumber = createIdGenerator();
+  const OBJ = [];
```

Comment:
Это изменяемый объект, нужно в camelCase нотации, например `photos` или `descriptions`, в крайнем случае можно оставить и `objects` главное во множественном числе, это же массив объектов будет

```suggestion
  const photos = [];
```

Reply:
+